tests/base_conftest.py

```diff
@@ -215,8 +215,8 @@ def get_logs(tx_hash, c, event_name):
 
 
 @pytest.fixture(scope="module")
-def assert_w3_tx_failed(tester):
-    def assert_w3_tx_failed(function_to_test, exception=TransactionFailed, exc_text=None):
+def w3_assert_tx_failed(tester):
+    def w3_assert_tx_failed(function_to_test, exception=TransactionFailed, exc_text=None):
```

Comment:
Why rename all of this?

Reply:
There are two sets of tests with this change that require different fixtures to set up the test cases - those that will use titanoboa, and those that will use w3 due to certain features such as block mining. Therefore, I renamed the existing w3 fixtures with a `w3` prefix, and switched to titanoboa for the existing fixture names.

Reply:
hmm. i think we can get rid of `assert_tx_failed` entirely and replace with uses of `boa.reverts()`. are there test cases which really depend on block mining / sealing?